src/Auth.php

```diff
@@ -27,25 +27,25 @@ public function __construct(Authentication $authenticate)
     }
 
     /**
-     * Sets the handler that should be used for this request.
+     * Sets the Authenticator alias that should be used for this request.
      */
-    public function setHandler(?string $handler = null)
+    public function setAuthenticator(?string $alias = null)
     {
-        if (! empty($handler)) {
-            $this->handler = $handler;
+        if (! empty($alias)) {
+            $this->alias = $alias;
         }
 
         return $this;
     }
 
     /**
-     * Returns the handler name.
+     * Returns the Authenticator alias.
      *
      * @return string
      */
-    public function getHandler()
+    public function getAlias()
```

Comment:
This method seems to be unused. Can we remove? @lonnieezell

Reply:
You're right that we don't currently use it. I'm trying to think of use cases where it might be handy and I'm drawing a blank currently. Unless you can think of a use case when extending it or daily usage, it's probably safe to delete.

Reply:
Removed.